.gitignore

```diff
@@ -1,7 +1,6 @@
 .idea
 .vscode
 node_modules
-dist
```

Comment:
This is a temporary change to be able to test this branch on projects with commit hash